float64 mass_position
float64 thrust
float64 heave_force
float64 roll_torque
float64 pitch_torque
